app/Providers/RouteServiceProvider.php

```diff
@@ -39,6 +39,10 @@ public function map()
 
         $this->mapWebRoutes();
 
+        $this->mapUserRoutes();
```

Comment:
what is use of Providers? can u explain. i am not aware

Reply:
sir, providers are generally used to configure the application and register various functionality including routing. so, here I am using it to register new route file.